app/scss/adminPage.scss

```diff
@@ -180,3 +180,6 @@ input[type="number"] {
   }
 }
 
+.categoryItem {
+  margin-bottom: 10px;
```

Comment:
can we have a bit more space? and maybe make the category names a bit bigger. they feel a bit randomly floaty at the moment. maybe try something like:
https://getbootstrap.com/docs/4.0/components/list-group/

Reply:
Make each category item have the same min height.